content/3_Silver/Prefix_Sums_2.problems.json

```diff
@@ -43,19 +43,6 @@
         "kind": "internal"
       }
     },
-    {
```

Comment:
Is this necessary?

Reply:
What do you mean?

Reply:
OHH just remembered: yea I deleted this during the merge cuz the website wasn't properly building locally.